### PR TITLE
Simplify the metrics examples

### PR DESCRIPTION
**Pull request by reyang**

### CODE REVIEW

File: docs/metrics/getting-started-gauge/Program.cs [hunk 1]
```diff
@@ -23,27 +23,21 @@
...
+            () => new List<Measurement<long>>()
             {
-                var tag1 = new KeyValuePair<string, object>("tag1", "value1");
-                var tag2 = new KeyValuePair<string, object>("tag2", "value2");
-
-                return new List<Measurement<long>>()
-                {
-                    new Measurement<long>(RandomGenerator.Next(1, 1000), tag1, tag2),
-                };
+                new (RandomGenerator.Next(1, 1000), new ("tag1", "value1"), new ("tag2", "value2")),
```
Comment (cijothomas) [inline docs/metrics/getting-started-gauge/Program.cs:40]: we may use `Process.GetCurrentProcess().WorkingSet64`

**Review by cijothomas — APPROVED** [review]

File: docs/metrics/getting-started-gauge/Program.cs (hunk 1 above)
Comment (reyang) [reply docs/metrics/getting-started-gauge/Program.cs:40]: Let me update that.

Comment (reyang) [reply docs/metrics/getting-started-gauge/Program.cs:40]: Just tried and it's pretty boring, the numbers remain the same (I guess it's partially due to the fact that CLR pre allocates).

Comment (cijothomas) [reply docs/metrics/getting-started-gauge/Program.cs:40]: Our sdk is good at *not* leaking memory ;)

File: docs/metrics/getting-started-observable-counter/README.md [hunk 2]
```diff
@@ -33,13 +33,13 @@ output from the console, similar to shown below:
 <!-- markdownlint-disable MD013 -->
 ```text
 Service.Nameunknown_service:getting-started-observable-counter
-Export 16:35:25.669 16:35:25.670 observable-counter [tag1=value1;tag2=value2] LongSum, Meter: TestMeter/0.0.1
+Export 16:35:25.669 16:35:25.670 observable-counter [tag1=value1;tag2=value2] LongSum, Meter: MyMeter/0.0.1
```
Comment (reyang) [inline docs/metrics/getting-started-observable-counter/README.md:36]: Some of the output is very outdated, and the console exporter needs to be fixed (seems there are merge issues that some changes were lost). Can be another PR.

File: docs/metrics/getting-started-observable-counter/README.md [hunk 3]
```diff
@@ -50,14 +50,14 @@ What does the above program do?
 
 The program creates a
 [Meter](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#meter)
-instance named "TestMeter" and then creates a
+instance named "MyMeter" and then creates a
```
Comment (cijothomas) [inline docs/metrics/getting-started-observable-counter/README.md:53]: the sentence below this need to be update as we are now reporting processortime, instead of just any other number.

Comment (reyang) [reply docs/metrics/getting-started-observable-counter/README.md:53]: I guess it's probably fine to not mention the details?

Comment (cijothomas) [reply docs/metrics/getting-started-observable-counter/README.md:53]: okay with me

**Review by cijothomas — APPROVED** [review]

**Comment by reyang** [comment]
It seems I've hit a catch-22 here - I can either make the `dotnet build` happy or make the `dotnet format` happy, but not both:
* without the following change, `dotnet format` would complain and asking for the spaces (and what's interesting is that it only asks for spaces in these two files)
* with the following change, `dotnet build` would fail due to warning `warning SA1000: The keyword 'new' should be followed by a space.`

image

**Review by cijothomas — APPROVED** [review]
